src/components/FileUpload.tsx

```diff
@@ -44,6 +44,7 @@ export type UploadedDatasets = {
 };
 
 export default function FileUpload() {
+    const { enqueueSnackbar } = useSnackbar();
```

Comment:
This is really cool, we should use it in the rest of the app

Reply:
Thanks